drivers/dai/intel/dmic/dmic.h

```diff
@@ -410,14 +410,22 @@
 #define DB2LIN_FIXED_INPUT_QY 24
 #define DB2LIN_FIXED_OUTPUT_QY 20
 
-/* Hardwired log ramp parameters. The first value is the initial gain in
- * decibels. The default ramp time is provided by 1st order equation
- * ramp time = coef * samplerate + offset. The default ramp is 200 ms for
- * 48 kHz and 400 ms for 16 kHz.
+/* Hardcoded log ramp parameters. The default ramp is 200 ms for 48 kHz
```

Comment:
was the comment to say "100ms for 48kHz and 200ms for 16kHz", or did I understand the PR description wrong?

Reply:
Oops, yes need to fix.

Reply:
Fixed now